src/Localization/en/SqlX.json

```diff
@@ -37,16 +37,20 @@
     "CannotSave": "Cannot save the changes to the Dedicated gateway resource at the moment.",
     "DedicatedGatewayEndpoint": "Dedicated gatewayEndpoint",
     "NoValue": "",
-    "SKUDetails": "SKU Details:",
     "CosmosD4Details": "General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory",
     "CosmosD8Details": "General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory",
     "CosmosD16Details": "General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory",
-    "CosmosD32Details": "General Purpose Cosmos Compute with 32 vCPUs, 128 GB Memory",
     "Cost": "Cost",
     "CostText": "Hourly cost of the dedicated gateway resource depends on the SKU selection, number of instances per region, and number of regions.",
     "ConnectionString": "Connection String",
     "ConnectionStringText": "To use the dedicated gateway, use the connection string shown in ",
-    "KeysBlade": "the keys blade",
+    "KeysBlade": "the keys blade.",
+    "MetricsString": "Metrics",
+    "MetricsText": "Monitor the \"DedicatedGatewayCpuUsage\" and \"DedicatedGatewayMemoryUsage\" in ",
+    "MetricsBlade": "the metrics blade.",
+    "ResizingDecisionText": "To understand if the dedicated gateway is the right size, ",
+    "ResizingDecisionLink": "learn more about dedicated gateway sizing.",
+    "NoChangesToExistingResource": "No changes were made to the existing resource. Saving will have no effect.",
```

Comment:
>No changes were made to the existin [](http://example.com/codeflow?start=36&length=35)

i htink based on Srinath's comment, you no longer need this. please follow up with him if needed.

Reply:
Yes, these need to be removed,